File: Detailed Differences/DetailedDifferences.java
import java.io.*;
public class DetailedDifferences {
    public static void main(String[] args) throws IOException {
        BufferedReader sc = new BufferedReader(new InputStreamReader(System.in));
        PrintWriter writer = new PrintWriter(System.out);
        int q = Integer.parseInt(sc.readLine());
        for (int i = 0;i < q;i++) {
            String a = sc.readLine();
            String b = sc.readLine();
            writer.println(a);
            writer.println(b);
            for (int j = 0; j < a.length(); j++) {
                if (a.charAt(j)==b.charAt(j)) {
                    writer.write('.');
                } else {
                    writer.write('*');
                }
            }
            writer.println();
        }
        writer.flush();
    }
}

File: Detailed Differences/detaileddifferences.js
var q = parseInt(readline());
for (let i = 0; i < q; i++) {
    let a = readline();
    let b = readline();
    let result = "";
    for (let j = 0; j < a.length; j++) {
        if (a[j]===b[j]) {
            result += ".";
        } else {
            result += "*";
        }
    }
    console.log(a);
    console.log(b);
    console.log(result);
}